js/views/file_input_view.js

```diff
@@ -281,6 +282,19 @@
             e.stopPropagation();
             e.preventDefault();
             this.$el.removeClass('dropoff');
+        },
+        onPaste: function(e) {
+            var items = e.originalEvent.clipboardData.items;
+            var imgBlob = null;
+            for (var i = 0; i < items.length; i++) {
```

Comment:
It's possible to copy content with multiple items of different types. e.g., right-click -> copy an image in chrome will add both the image and its url to the clipboard.